ios/LiveKitReactNativeModule.swift

```diff
@@ -70,17 +70,39 @@ public class LivekitReactNativeModule: RCTEventEmitter {
         }
         RTCAudioSessionConfiguration.setWebRTC(rtcConfig)
     }
-    
-    @objc(startAudioSession)
-    public func startAudioSession() {
-        // intentionally left empty
+
+    @objc(startAudioSession:withRejecter:)
+    public func startAudioSession(resolve: RCTPromiseResolveBlock, reject: RCTPromiseRejectBlock) {
+        let session = RTCAudioSession.sharedInstance()
+        session.lockForConfiguration()
+        defer {
+            session.unlockForConfiguration()
+        }
+
+        do {
+            try session.setActive(true)
```

Comment:
Moved `setActive(true)` here, I haven't confirmed it this get's invoked before starting internal AVAudioEngine, I assume it does but need to double check.